exporter/exporterhelper/metricshelper.go

```diff
@@ -29,18 +29,18 @@ import (
 type PushMetricsData func(ctx context.Context, td consumerdata.MetricsData) (droppedMetrics int, err error)
 
 type metricsExporter struct {
-	exporterFormat  string
+	exporterName    string
 	pushMetricsData PushMetricsData
 }
 
 var _ (exporter.MetricsExporter) = (*metricsExporter)(nil)
 
 func (me *metricsExporter) MetricsExportFormat() string {
```

Comment:
Should we also rename this function?

Reply:
Yes, however, the code is not implementing it consistently. Currently the function is only used by tests. I will review its usage and will fix in a separate PR.